Makefile

```diff
@@ -1,5 +0,0 @@
-# will add the license headers to all of our files
-# requires go installed in your system: https://golang.org/doc/install
-addheaders:
```

Comment:
This is actually pretty useful. We can add the install command to install this binary and make it OS-independent.

Reply:
Must it be at the repository's root, though? Can we remove it for now and add the necessary command with a dedicated PR?

Reply:
Can you leave this file here so I can address this in a new PR?

Reply:
Sure! Will re-add!